### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by andrianarivo**
The things I did in this PR:

1. Properly configure Linters
2. Use Githubflow
3. Modularize architecture
4. Add a new Task
5. Remove an existing Task
6. Edit an existing Task

### CODE REVIEW

File: src/modules/Storage.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const getTasks = () => JSON.parse(localStorage.getItem('tasks')) || [];
+
+const saveTasks = (tasksArray) => {
+  localStorage.setItem('tasks', JSON.stringify(tasksArray));
+};
```
Comment (1GORDON) [inline src/modules/Storage.js:5]: Great design @andrianarivo 🤝

- [ ] As stated in the project requirements, kindly ensure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). This is not the case for your design. Kindly work on this and submit your project for approval.
Indexes

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @andrianarivo ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Storage.js (hunk 1 above)
Comment (andrianarivo) [reply src/modules/Storage.js:5]: @1GORDON I need more clarifications, are you able to join a slack call ?

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — COMMENTED** [review]
Hello @andrianarivo 👋,

_This is an additional review as a result of the second opinion request._

It seems you have understood the suggested change by the reviewer and have even pushed changes on this PR to that effect. Please ask for another code review. The next reviewer will check your work and see if you've addressed the concerns from the first review.

Keep rocking 🚀!

**Pull request merged**